api/app.js

```diff
@@ -55,6 +57,8 @@ app.use(cookieParser());
 // application routes
 app.use('/', indexRouter);
 app.use(['/profile', '/profiles'], profileRouter);
+app.use(['/animal', '/animals'], animalRouter);
+app.use(['/service', '/services'], serviceRouter);
```

Comment:
I like that there are multiple options for these url endpoints. Especially right now while we're deep into production right now.